lib/service/storage/queryTranslator.js

```diff
@@ -68,7 +68,7 @@ const KONCORDE_CLAUSES_TO_ES = {
     }
   }),
   in: content => ({
-    term: {
+    terms: {
```

Comment:
Wouldn't this be a breaking change? (given that for a time, **term** was the usable 'term')

Reply:
To me it's not a breaking change, since the external API doesn't change. Rather, it's a bug fix since a `in` term in Koncorde allows to match multiple values, which needs the `terms` key in ES queries (`term` only allows matching a single value)